organisations/factories.py

```diff
@@ -3,6 +3,13 @@
 from organisations.models import Organisation, OrganisationProposal, Person
 
 
+def fake_id_array():
+    from faker import Faker
+
+    fake = Faker()
+    return fake.pylist(3, True, "str")
```

Comment:
Hmm, what keeps these values the same between test runs (for snapshots)?

Reply:
The seed configuration, I guess(?).
```
@pytest.fixture(autouse=True)
def setup_test_environment(settings):
    settings.CAPTCHA_ENABLED = False
    factory.random.reseed_random("777")
    with freeze_time("2020-01-04"):
        yield
```

I admit that I would need to get to know the subject a little more closely myself. I’m going to do it as soon as I get time on the subject. Sometimes we need to update snapshots that are not even remotely related to the updated code and I would like to know why. So far I have blamed it in part on Graphene’s fault

Reply:
If it would change every time, Github would not pass my snapshot 😃